scalding-repl/src/main/scala/com/twitter/scalding/ReplImplicits.scala

```diff
@@ -26,6 +26,9 @@ import scala.concurrent.{ Future, ExecutionContext => ConcurrentExecutionContext
  * Most of these conversions come from the [[com.twitter.scalding.Job]] class.
  */
 object ReplImplicits extends FieldConversions {
+  val mr1Key = "mapred.job.tracker"
```

Comment:
can these fields be private? ReplImplicits is imported by everyone using the REPL.